go.mod

```diff
@@ -63,7 +63,7 @@ require (
 	github.com/grafana/cuetsy v0.1.11 // @grafana/grafana-as-code
 	github.com/grafana/grafana-aws-sdk v0.23.1 // @grafana/aws-datasources
 	github.com/grafana/grafana-azure-sdk-go v1.12.0 // @grafana/partner-datasources
-	github.com/grafana/grafana-plugin-sdk-go v0.208.0 // @grafana/plugins-platform-backend
+	github.com/grafana/grafana-plugin-sdk-go v0.209.1-0.20240208231327-bb7c8753e380 // @grafana/plugins-platform-backend
```

Comment:
^^ will be replaced with a real version after the PR is merged

Reply:
When the PR is merged you mean?

Reply:
This PR https://github.com/grafana/grafana-plugin-sdk-go/pull/890

Reply:
[v0.210.0 released](https://github.com/grafana/grafana-plugin-sdk-go/releases/tag/v0.210.0), you can update this.

Reply:
We will do it on Tuesday